docs/troubleshooting.md

```diff
@@ -174,7 +174,7 @@ cabal update && cabal install :pkg:haskell-language-server
 If you are compiling with a ghc version with a specific `cabal-ghc${ghcVersion}.project` in the repo you will have to use it. For example for ghc-9.0.x:
 
 ```bash
-cabal update && cabal install :pkg:haskell-language-server --project-file=cabal-ghc90.project
+cabal update && cabal install pkg:haskell-language-server
```

Comment:
Why did you change this line, it is now incoherent with the above documentation which states that "If you are compiling with a ghc version with a specific cabal...project".

Reply:
Came with @pepeiborra changes

Reply:
resolved now that @pepeiborra has merged his changes in master branch